### DISCUSSION TITLE
Can the Slack action support threading (replying to an existing message)?

### DISCUSSION THREAD

**Discussion by richellis** [fastlane/fastlane#16921] [Ideas]
The Jenkins CI slack-plugin has [thread support](https://github.com/jenkinsci/slack-plugin#threads-support) by simply passing a `threadId` to the `channel` parameter. This would be great to have from fastlane as well. I looked at the code a bit but I'm no rubyist so I'm not sure if the [slack-notifier](https://github.com/stevenosloan/slack-notifier/blob/master/lib/slack-notifier.rb) dependency that fastlane uses (at least I think that's the one it uses) already supports that or if it would "just work".

To be useful I think the slack action would also need a response object of some kind to get the `threadId` from an initial message, which could then be used to send subsequent messages as threaded replies. (I'm also not sure whether the underlying slack-notifier `ping` method already has that, either.)

**Comment by crazymanish** [comment]
Hi @richellis 

You can use below Fastlane plugin to support slack messages inside the thread 
https://github.com/crazymanish/fastlane-plugin-slack_bot
